modules/distribution-workflowoperation/src/main/java/org/opencastproject/workflow/handler/distribution/ConfigurablePublishWorkflowOperationHandler.java

```diff
@@ -126,6 +138,37 @@ protected DownloadDistributionService getDistributionService() {
     return distributionService;
   }
 
+  private Map<String, Object> addMetadataCatalogsToTemplateData(Map<String, Object> values, MediaPackage mp) {
+    if (! mp.hasCatalogs()) {
+      logger.info("No metdata catalogs found.");
+      return values;
+    }
+    if (values == null) values = new HashMap<>();
+
+    for (Catalog catalog : mp.getCatalogs()) {
+      if (catalog.getFlavor() != null) {
+        String mainflavor = catalog.getFlavor().getType();
+        String subflavor = catalog.getFlavor().getSubtype();
+        Map<String, Object> metadata = new HashMap<>();
+        try {
+          DublinCoreCatalog dc = DublinCoreUtil.loadDublinCore(workspace, catalog);
+          for (Map.Entry<EName, List<DublinCoreValue>> entry : dc.getValues().entrySet()) {
+            String value = entry.getValue().get(0).getValue();
+            metadata.put(entry.getKey().getLocalName(), value);
+          }
```

Comment:
This will ignore namespaces and basically put a random element – the first value of the key in the last namespace – in the `Map`. This is okay to do, but should nbe mentioned in the documentation.